src/Rules/PackageData.hs

```diff
@@ -3,115 +3,32 @@ module Rules.PackageData (buildPackageData) where
 import Base
 import Context
 import Expression
-import Oracles.Setting
-import Rules.Generate
+import GHC.Packages
 import Settings.Packages.Rts
 import Target
 import Utilities
 
+import Hadrian.Haskell.Cabal.Parse (configurePackage)
+
 -- | Build @package-data.mk@ by using ghc-cabal utility to process .cabal files.
 buildPackageData :: Context -> Rules ()
```

Comment:
Shall this module and this `Rules` set be renamed? Also, please fix the comment.

Reply:
Updated the comment in an upcoming commit.